### PR TITLE
Feature/day 8

### PR DESCRIPTION
**Pull request by tejaschapke18**

### CODE REVIEW

File: Node-Backend/.env
Comment (Sandip-rana-successive) [inline Node-Backend/.env:1]: add .env file to .gitignore, do not push it to the cloud

File: Node-Backend/dist/Controllers/countryController.js
Comment (Sandip-rana-successive) [inline Node-Backend/dist/Controllers/countryController.js:1]: don't push dist folder on git

File: Node-Backend/dist/Controllers/countryDataSeed.js
Comment (Sandip-rana-successive) [inline Node-Backend/dist/Controllers/countryDataSeed.js:1]: same as above

File: Node-Backend/src/app.ts [hunk 1]
```diff
@@ -20,6 +23,20 @@ app.use(logRequest);
 app.use("/api", userRouter);
 app.use(errorHandler);
 
+const seedDB = async () => {
```
Comment (Sandip-rana-successive) [inline Node-Backend/src/app.ts:26]: Could you please explain seedDB funciton's functionality ?

**Review by Sandip-rana-successive — CHANGES_REQUESTED** [review]
Please fix the feedback and update the PR

File: Node-Backend/.env
Comment (tejaschapke18) [reply Node-Backend/.env:1]: Done, removed .env

File: Node-Backend/dist/Controllers/countryController.js
Comment (tejaschapke18) [reply Node-Backend/dist/Controllers/countryController.js:1]: Done removed dist folder

File: Node-Backend/src/app.ts (hunk 1 above)
Comment (tejaschapke18) [reply Node-Backend/src/app.ts:26]: In this firstly removing all data and inserting data.
But in next day PR I have changed it : inserting data only once.

Comment (Sandip-rana-successive) [reply Node-Backend/src/app.ts:26]: okay

**Review by Sandip-rana-successive — APPROVED** [review]

**Pull request merged**